From 62ffb7dcde3da0e152906e63bd945d7eddfb320a Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Sat, 24 Nov 2018 00:45:00 +0100
Subject: [PATCH] python3: bump version to 3.6.7

There is problem with compilation with OpenSSL 1.1.0f on our build
servers. There seems to be some fixes for OpenSSL 1.1.0 later revisions
in Python 3.6.6 so let's test newest version.
---
 lang/python/python3-version.mk | 4 ++--
 lang/python/python3/Makefile   | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/lang/python/python3-version.mk b/lang/python/python3-version.mk
index 522971758..1c7dc46b3 100644
--- a/lang/python/python3-version.mk
+++ b/lang/python/python3-version.mk
@@ -8,7 +8,7 @@
 # Note: keep in sync with setuptools & pip
 PYTHON3_VERSION_MAJOR:=3
 PYTHON3_VERSION_MINOR:=6
-PYTHON3_VERSION_MICRO:=5
+PYTHON3_VERSION_MICRO:=7
 
 PYTHON3_VERSION:=$(PYTHON3_VERSION_MAJOR).$(PYTHON3_VERSION_MINOR)
 
@@ -16,4 +16,4 @@ PYTHON3_SETUPTOOLS_PKG_RELEASE:=2
 PYTHON3_PIP_PKG_RELEASE:=2
 
 PYTHON3_SETUPTOOLS_VERSION:=39.0.1
-PYTHON3_PIP_VERSION:=9.0.3
+PYTHON3_PIP_VERSION:=10.0.1
diff --git a/lang/python/python3/Makefile b/lang/python/python3/Makefile
index 304e2e51a..f419b3a8b 100644
--- a/lang/python/python3/Makefile
+++ b/lang/python/python3/Makefile
@@ -19,7 +19,7 @@ PKG_VERSION:=$(PYTHON_VERSION).$(PYTHON_VERSION_MICRO)
 
 PKG_SOURCE:=Python-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://www.python.org/ftp/python/$(PKG_VERSION)
-PKG_HASH:=f434053ba1b5c8a5cc597e966ead3c5143012af827fd3f0697d21450bb8d87a6
+PKG_HASH:=81fd1401a9d66533b0a3e9e3f4ea1c7c6702d57d5b90d659f971e6f1b745f77d
 
 PKG_LICENSE:=PSF
 PKG_LICENSE_FILES:=LICENSE Modules/_ctypes/libffi_msvc/LICENSE Modules/_ctypes/darwin/LICENSE Modules/_ctypes/libffi/LICENSE Modules/_ctypes/libffi_osx/LICENSE Tools/pybench/LICENSE
-- 
2.20.1

